scrape_bonds.py

```
@@ -0,0 +1,298 @@
+        t = t[:-1]
+    elif t.endswith('B'):
+        mult = 1000.0
+        t = t[:-1]
+    elif t.endswith('K'):
+        mult = 0.001
+        t = t[:-1]
+    # Now attempt float
+    try:
+        val = float(t)
+        return round(val * mult, 6)
+    except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)